### ISSUE TITLE
Bug with ion-select style (label position = floating)?

### ISSUE THREAD

**Issue by ramax495** [ngx-formly/ngx-formly#2635]
I use formly for ionic and have a problem with control of type 'select' with labelPosition: 'floating'.
ion-select has css property: 

```css
:host(.in-item) {
    position: static;
    max-width: 45%;
}
```
Because of 'max-width: 45%' selected value is not fully visible - screenshot http://nimb.ws/i4z23B.
I tried different ways but couldn't customize it.
Is it bug ? How can I change css property 'max-width'?

Sample: https://stackblitz.com/edit/angular-ionic-formly-select?file=src/app/app.component.ts

**Comment by ramax495** [comment]
As it turned out this is a Formly library bug.
_formly-field-select-ion_ tag is inserted between _ion-item_ and ion-select _ionic tags_. And it crashes ionic styles (in case we use position floating option for label).

Solution is adding additional css in component styles file:

```css
::ng-deep .item-label-floating ion-select {
  align-self: stretch;
  max-width: 100%;
}
```

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [5.10.11](https://github.com/ngx-formly/ngx-formly/releases/tag/5.10.11) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
